os/fs/driver/mtd/Kconfig

```diff
@@ -229,6 +229,31 @@ config SMART_CRC_16
 
 endchoice
 
+config MTD_SMART_JOURNALING
+    bool "Enable filesystem journaling for smartfs"
+	default n
+   	select MTD_SMART_ENABLE_CRC
+	---help---
+		Enables logging of write operations on filesystem, so that in
+		case of power failure the filesystem can be brought to a stable
+		state with minimal data loss.
+		Enabling journaling will increase the delay in filesystem
+		operations, because it write journal data before it commit sector.
+		It uses CRC-16 so please enable SMART_CRC_16
+
+if MTD_SMART_JOURNALING
+
+config MTD_SMART_JOURNALING_NSECTOR
+	int "Number of reserved logical sector for Journal"
+	default 16
+	---help---
+		Journal sector will be reserved right after smart MTD initialize,
+		It will divided to 2 area to toggle active area.
+		Journal has it's sequence to varify itself.
```

Comment:
Is varfiy the right word?

Reply:
yes it means that 'To make  different'

Reply:
I think varify -> verify?

Reply:
ok I think this makes confusing, changed comment. please verify grammer and something more.